lib/gasman.gd

```diff
@@ -49,6 +49,25 @@
 ##
 DeclareGlobalFunction("CollectGarbage");
 
+#############################################################################
+##
+#F  GasmanDisableMemoryLimits( <full> )
+##
+##  <#GAPDoc Label="GasmanDisableMemoryLimits">
+##  <ManSection>
+##  <Func Name="GasmanDisableMemoryLimits" Arg=''/>
+##  <Returns>
+##  nothing.
+##  </Returns>
+##  <Description>
+##  Disables limits on the size of GAP's workspace. These are usually
+##  set using the --maxworkspace and --limitworkspace command line options.
```

Comment:
Wait, I didn't know about `--limitworkspace` ?!? Grepping for it reveals one hit in `lib/system.g`, where it is an alias for `-K` . But in `system.c`, we set `--maximal-workspace` as alias for `-K`. That seems to be a bug, dating back to commit 2b115dc?

Reply:
Will check this